Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -245,11 +248,11 @@ - (void)commitMutations:(NSArray<FSTMutation *> *)mutations
   [self invokeRPCWithFactory:rpcFactory errorHandler:completion];
 }
 
-- (void)lookupDocuments:(NSArray<FSTDocumentKey *> *)keys
+- (void)lookupDocuments:(const std::vector<DocumentKey>)keys
```

Comment:
Was this intended to be a const reference?

Reply:
No, `keys` is used in block and thus cannot make it a reference (without make an extra copy).

Reply:
Can an additional variable be used?
```cpp
__block const auto& keys_ref = keys;
// or
__block const auto* keys_ptr = &keys;
```
? Sorry if that's inapplicable, my Objective-C is weak.